app/controllers/manuals_controller.rb

```diff
@@ -1,6 +1,11 @@
 class ManualsController < ApplicationController
-  before_action :ensure_manual_is_found
-  before_action :ensure_document_is_found, only: :show
```

Comment:
Are we losing some functionality here?

Reply:
I don't think so as the functionality these callbacks provided are now represented by the `follow_redirect_if_document_redirected` callback and the [rescue blocks](https://github.com/alphagov/manuals-frontend/pull/1113/files#diff-6b5ff7cba3fc3659ddf2782828d61b92542869a958e4970b0d650d108ba881c0R2-R4). These were previously handled by [rescuing](https://github.com/alphagov/manuals-frontend/pull/1113/files#diff-6b5ff7cba3fc3659ddf2782828d61b92542869a958e4970b0d650d108ba881c0L75-L77) at a different point and comparing [nils](https://github.com/alphagov/manuals-frontend/pull/1113/files#diff-6b5ff7cba3fc3659ddf2782828d61b92542869a958e4970b0d650d108ba881c0L42). 

The rescue classes are different than what was previously implemented but they mean the same thing. E.g in [gds-api-adapters](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/content_store.rb#L7) the `GdsApi::ContentStore::ItemNotFound` error class inherits from `GdsApi::HTTPNotFound`.